docs/index.rst

```diff
@@ -9,24 +9,11 @@ Features
 
 .. image:: screenshot2.png
 
-Installation
+Quick Installation
 ------------
 
-Install from PyPI the backend specific variant of django-constance:
-
-For the (default) Redis backend::
-
     pip install "django-constance[redis]"
 
-For the database backend::
-
-    pip install "django-constance[database]"
```

Comment:
You have completely excluded information about `database` backend. This is not a good approach.
Probably there really is some kind of problem with the documentation in this aspect, but just not mentioning the `database` backend is not a solution.

Reply:
I personally use the `database` backend at work and when I started I was looking for some configuration instructions. I believe there are many more users who also need it. So I agree with @Mogost it definitely shouldn't be removed from the docs.

Reply:
I also use the `database` backend. I think there's some confusion here - *this pull request doesn't completely remove information about the `database` backend from the documentation*, it removes *incomplete* information that causes people trying to use the `database` backend to run into problems.

I think adding a link to the Backends page (which contains a complete set of installation instructions for the `database` backend) to the Installation section should assuage the concerns raised here.

Technically, updating the Installation section to include complete instructions would also accomplish the same goal. However, since this would duplicate the information on the Backends page, I don't think that would be the best solution.